### PR TITLE
disables reset

### PR DESCRIPTION
**Pull request by merrickfox**
@kulor I think the dependency in b&w for pikaday points to a specific commit, so we might have to merge this and your modifications into master and use that?

### CODE REVIEW

File: pikaday.js [hunk 1]
```diff
@@ -1042,9 +1042,11 @@
                 if (this._o.bound) {
                     removeEvent(document, 'click', this._onClick);
                 }
-                this.el.style.position = 'static'; // reset
-                this.el.style.left = 'auto';
-                this.el.style.top = 'auto';
+                // disables reset to prevent bug on b&w.com
```
Comment (merrickfox) [inline pikaday.js]: Let me know if you want me to just straight up remove these lines

Comment (kulor) [inline pikaday.js]: Yep, just remove them

Comment (merrickfox) [inline pikaday.js]: Done

**Comment by kulor** [comment]
Re:dependency, we should make use of tags/releases in Github :)

**Pull request merged**